tests/web_api/test_spec.py

```diff
@@ -42,7 +42,7 @@ def test_entity_definition():
     assert_in('children', dpath.get(body, 'definitions/Household/properties'))
     assert_in('salary', dpath.get(body, 'definitions/Person/properties'))
     assert_in('rent', dpath.get(body, 'definitions/Household/properties'))
-    assert_equal('number', dpath.get(body, 'definitions/Person/properties/salary/additionalProperties/type'))
+    assert 'number', dpath.get(body == 'definitions/Person/properties/salary/additionalProperties/type')
```

Comment:
Whoa, nope. This transformation does not do what you think it does.

I'm surprised this test even passes with this change… I think I see why: what comes after `assert` here is in fact a tuple and considered truthy by Python.

Reply:
Good catch, thanks! Abusive use of regexps...